src/Sulu/Component/Content/Document/Subscriber/OrderSubscriber.php

```diff
@@ -64,19 +73,18 @@ public function handlePersist(PersistEvent $event)
 
         $propertyName = $this->encoder->systemName(self::FIELD);
 
-        if ($node->hasProperty($propertyName)) {
-            return;
-        }
-
         $parent = $node->getParent();
-        $nodeCount = count($parent->getNodes());
-        $order = ($nodeCount + 1) * 10;
+        $count = 0;
+        foreach ($parent->getNodes() as $childNode) {
```

Comment:
@danrot ... should this not only be done while changing the order or moving the node ... looks to me like a overhead, if it's done for each persist?